src/main/java/org/dimdev/riftloader/RiftLoader.java

```diff
@@ -38,6 +38,7 @@
     private InstanceListMap listeners = new InstanceListMap();
     private InstanceListMap customListenerInstances = new InstanceListMap();
 
+    @SuppressWarnings("unused") //called via reflection
```

Comment:
Why @SuppressWarnings("unused")? Public methods don't produce "unused" warnings unless you changed the default IntelliJ inspection settings.